### PR TITLE
Feature/shopping list

### PR DESCRIPTION
**Pull request by sainumtown**
Please review

### CODE REVIEW

File: app/src/main/java/com/padc/recipes/adapters/ShoppingListAdapter.java [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+        for (int i = 0; i < materiaLArray.length; i++) {
+            MMCheckBox cbMaterial = new MMCheckBox(RecipesApp.getContext());
+            cbMaterial.setTextColor(view.getResources().getColor(R.color.text_black_ish));
+            cbMaterial.setButtonDrawable(R.drawable.checkbox);
+             int p = R.dimen.padding_medium;
+            cbMaterial.setPadding(10,10,50,10);
+            cbMaterial.setText(materiaLArray[i]);
+
+            final int finalI = i;
+            cbMaterial.setOnClickListener(new View.OnClickListener() {
```
Comment (vincent-paing) [inline app/src/main/java/com/padc/recipes/adapters/ShoppingListAdapter.java]: You have to use setOnCheckedChangeListener for CheckBox. Or if you're planning to enable selecting one, Use RadioButton

Comment (sainumtown) [reply app/src/main/java/com/padc/recipes/adapters/ShoppingListAdapter.java]: We will use multiple checkbox.
Updated to on Check listener.

Comment (sainumtown) [reply app/src/main/java/com/padc/recipes/adapters/ShoppingListAdapter.java]: Finished resolving

**Comment by vincent-paing** [comment]
Forgot to add : This is because when you call setChecked(); on item. OnClickListener won't be called. It only calls OnCheckedChangeListener Interface if added

**Pull request merged**